intro-HPC/examples/SCOOP/picalc.py

```diff
@@ -29,10 +29,14 @@ def calcPi(workers, tries):
     nr_batches = 3000
     batch_size = 5000
 
-    try:
-        nr_batches = int(sys.argv[1])
-        batch_size = int(sys.argv[2])
-    except:
-        pass
-
-    print "PI=%f (in nr_batches=%d,batch_size=%d)" % (calcPi(nr_batches, batch_size), nr_batches, batch_size)
+    # Program name and two arguments
+    if len(sys.argv) == 3:
```

Comment:
What if only a single number is given? That'll result in using the default, which may be very confusing...